src/pages/export-page/ExportPage.tsx

```diff
@@ -0,0 +1,117 @@
+import { useMemo } from "react";
+import { Link } from "react-router-dom";
+
+import { Button } from "@lib/gds-components";
+import {
+  resolveDevelopmentPlanCSV,
+  resolveTimetableEventsCSV,
+} from "@lib/utils/timetable";
+import { PlanViewer } from "@lib/timetable-visualisation/PlanViewer";
+import { useFormContext } from "../../context/use-form-context";
+
+export const ExportPage = () => {
+  const { developmentPlan, timetableEvents } = useFormContext();
+
+  const developmentPlanDownloadLink = useMemo(() => {
+    const timetableCSV = resolveDevelopmentPlanCSV(
+      developmentPlan,
+      null // TODO: Replace with loaded development plan in update journey
+    );
+
+    return `data:text/csv;charset=urf-8,${timetableCSV}`;
+  }, [developmentPlan]);
+
+  const timetableEventsDownloadLink = useMemo(() => {
+    const timetableCSV = resolveTimetableEventsCSV(
+      timetableEvents,
+      null // TODO: Replace with loaded timetable events in update journey
+    );
+
+    return `data:text/csv;charset=urf-8,${timetableCSV}`;
+  }, [timetableEvents]);
+
+  return (
+    <>
+      <h1 className="govuk-heading-xl govuk-!-margin-top-6">
+        Export your timetable
+      </h1>
+
+      <p className="govuk-body">
+        When you have completed all the details, you’ll need to export your data
+        as CSV files. You can use these files to generate the timetable on your
+        website.
+      </p>
+
+      <p className="govuk-body">
+        <strong>Step 1</strong>
+      </p>
+
+      <p className="govuk-body">
+        You need to export <strong>both</strong> CSV files.
+      </p>
+
+      <p className="govuk-body">
+        <strong>Step 2</strong>
+      </p>
+
+      <p className="govuk-body">
+        Next, you’ll need to follow the guidance which explains how to use the
+        CSV files you’ve just downloaded to publish the timetable on your
+        website:{" "}
+        <Link
+          to="#" /* Replace with path to real page once it's been created */
+        >
+          Publishing a local plan timetable online: guidance for web teams
+        </Link>
+      </p>
+
+      <div>
```

Comment:
do we need these divs around the links?

Reply:
The extra divs stop the buttons displaying side by side